webanno-model/src/main/java/de/tudarmstadt/ukp/clarin/webanno/model/AnnotationLayer.java

```diff
@@ -100,6 +100,9 @@
     private boolean allowStacking;
 
     private boolean crossSentence;
+    
+    @Column(name = "showHover", nullable = false)
```

Comment:
See comment on non-nullable columns above.

Reply:
Changed in e95dfa245b637229bc930ccceedab8dc598c6521